### PR TITLE
Simplify task execution

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Minor tweaks to job execution connection management to simplify reuse of the `execute_job` function for https://github.com/learningequality/kolibri-installer-android/issues/104
* Makes the job storage path into an option so it can be configured via env vars, which lets the tests set the environment variable value for it to allow the multiprocessing environment to be properly configured without manually passing in the values through the worker execution
* Moves utility functions from main to utils as cleanup

## Reviewer guidance
Do all the tests still pass?

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/utils/tests/helpers.py [hunk 1]
```diff
@@ -21,7 +24,17 @@ def __init__(self, section, key, value):
 
     def enable(self):
         self.old_value = conf.OPTIONS[self.section][self.key]
+        envvars = option_spec[self.section][self.key]["envvars"]
+        self.old_envvars = {}
+        for envvar in envvars:
+            self.old_envvars[envvar] = os.environ.get(envvar, None)
+            os.environ[envvar] = self.temp_value
```
Comment (jredrejo) [inline kolibri/utils/tests/helpers.py]: `os.environ` are strings, it should be 
`os.environ[envvar] = str(self.temp_value)`
This is making some tests fail

**Review by jredrejo — CHANGES_REQUESTED** [review]

**Review by vkWeb — APPROVED** [review]
Code looks good to me! Tasks are working. 

Tests will surely pass after implementing @jredrejo's advice. I've checked it locally, coercing to `str` passes the tests.

File: kolibri/utils/tests/helpers.py (hunk 1 above)
Comment (rtibbles) [reply kolibri/utils/tests/helpers.py]: Ah yep - good catch!

Comment (rtibbles) [reply kolibri/utils/tests/helpers.py]: Updated!

**Review by jredrejo — APPROVED** [review]

**Pull request merged**